Unable to compile translated Markdown example with Typescript

Hi, I am trying get the [markdown example](https://github.com/yaronn/blessed-contrib/blob/master/examples/markdown.js) working in Typescript, but I struggle.

I have translated the example like so:
```typescript
import * as blessed from 'blessed';
import * as contrib from 'blessed-contrib';
import chalk from 'chalk';

const screen = blessed.screen();
const markdown = contrib.markdown();

screen.append(markdown);
markdown.setOptions({ firstHeading: chalk.red.italic });
markdown.setMarkdown('# Hello \n This is **markdown** printed in the `terminal` 11');
screen.render();
```
It produces:
```
⨯ Unable to compile TypeScript:
hello-markdown.ts(9,10): error TS2339: Property 'setOptions' does not exist on type 'MarkdownElement'.
hello-markdown.ts(10,10): error TS2339: Property 'setMarkdown' does not exist on type 'MarkdownElement'.
```

As I am fairly new to Typescript it is entirely possible that the problem has to do with how I translated or setup the example. Any pointers are much appreciated!

I created a demo to reproduce the problem like so:
```bash
git clone git@github.com:mamachanko/blessed-markdown-typescript-demo.git
cd blessed-markdown-typescript-demo
./run.sh . # builds and runs 'hello-markdown.ts' inside a docker image
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -358,6 +358,10 @@ declare namespace BlessedContrib {
             constructor(opts: MarkdownOptions)
 
             options: MarkdownOptions;
+
+            setOptions(options: any): void;
+
+            setMarkdown(markdown: string): void;
         }
 
 

